src/main/java/com/example/demo/repository/UserRepository.java

```diff
@@ -0,0 +1,28 @@
+package com.example.demo.repository;
+
+
+import java.util.List;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import com.example.demo.model.User;
+
+public interface UserRepository extends JpaRepository<User,Integer> {
+	
+	List<User> findByNameIgnoreCase(String emp_name);
+	
```

Comment:
Replace emp_name with just "name" in all the methods

Reply:
Done , though these methods are not being used anymore. So removing them